frontend/src/Router.jsx

```diff
@@ -30,6 +32,7 @@ const Router = () => {
         <Route path="/notifications" element={<Notifications />} />
         <Route path="/settings" element={<ProfileSettings />} />
         <Route path="/new-post" element={<PostComposer />} />
+        <Route path="/SeachPage" element={<SearchPage/>} />
```

Comment:
Watch out for the typo here - says "SeachPage"

Reply:
path names are kebab case for this one